doc/docker.md

```diff
@@ -33,9 +33,10 @@ The following are optional:
   of files downloaded from the server. This option is either `0` or `1`. The
   default is `0`.
 * `GEOIPUPDATE_VERBOSE` - Enable verbose mode. Prints out the steps that
-  `geoipupdate` takes. Set to **anything** (e.g., `1`) to enable.
-* `GEOIPUPDATE_CONF_FILE` - The path where the configuration file will be
-  written. The default is `/etc/GeoIP.conf`.
+  `geoipupdate` takes. Set to `1` to enable.
+* `GEOIPUPDATE_CONF_FILE` - The path of a configuration file to be used by
+  `geoipupdate`. If this file does not exist, an empty file will be created.
```

Comment:
Saying an empty file will be written seems odd, especially with no explanation. Maybe we can say something about it being optional. Actually the behaviour is going to be different now given the config file can be read now rather than written. We should clarify how this works and say something about it in the changelog I think. We could potentially just not take `GEOIPUPDATE_CONF_FILE` as an env var generally and remove it from this doc.